rust/src/linalg/cosine.rs

```diff
@@ -92,6 +92,8 @@ pub fn cosine_distance_batch(from: &[f32], to: &[f32], dimension: usize) -> Arc<
 
 #[cfg(target_arch = "x86_64")]
 mod x86_64 {
+    use super::dotprod_scalar;
+    use super::sq_scalar;
     use std::arch::x86_64::*;
```

Comment:
sort import by sections? `std / third party / lance` with a blank line in between.

Reply:
fixed